Update validator guide (#295)
w3f/polkadot-wiki: docs/maintain-guides-how-to-validate-kusama.md

brew install cmake pkg-config openssl git llvm
```

> ### Install & Configure NTP(Network Time Protocol) Client

Reviewer: ```suggestion
### Install & Configure Network Time Protocol (NTP) Client
```

> [NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) is a networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required the validators local clocks reasonably in sync, so you can check whether you have the NTP client by running: 

Reviewer 2: ```suggestion
[NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) is a networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service.  You can check whether you have the NTP client by running: 
```

_If you are using Ubuntu 18.04 / 19.04, NTP Client should be installed by default._

``` sh
timedatectl
```

> When you see something like `System clock synchronized: yes` then it is fine. If you do not see it, just install it by executing:

Reviewer 2: ```suggestion
If NTP is installed and running, you should see `System clock synchronized: yes` (or a similar message). If you do not see it, you can install it by executing:
```

Reviewer 2: We really should try to avoid words like "just", "simply", etc.

**Note:** The `--pruning=archive` flag is implied by the `--validator` and `--sentry` flags, so it is only required explicitly if you start your node without one of these two options. If you do not set your pruning to archive node, even when not running in validator and sentry mode, you will need to re-sync your database when you switch.

#### Previous Kusama CC2 Validator

Before synchronizing the latest chain data, you will need to copy your previous keystore to the new chain id in order to use the previously set session keys. If you do not do this, you will need to generate and set your session keys again.

> Kusama CC1 and Kusama CC2 have different default data directories, usually located in the `$HOME/.local/share/polkadot/chains` directory on Linux based machines. For example, the default directory for Kusama CC2 is `$HOME/.local/share/polkadot/chains/ksmcc2/keystore` while CC2 keys are located in `$HOME/.local/share/polkadot/chains/ksmcc2/keystore`.

Reviewer 2: Do we really still need to go into depth about the CC1 and CC2 chains?  We've been using CC3 since at least October/November.  Are nodes still going to be upgrading from them?

Author: We can remove that but we could imagine once Polkadot launch, validators will be also doing the same thing, so I guess it would be worth to stay here for reference.

Reviewer: Let's move it to it's own page on the Kusama guide? That way it's not directly in the guide but can still be accessible.

Author: sounds good

Click "Validate".

> If you go to the Staking tab, you should see a list of active validators currently running on the network. At the top of the page, it shows how many validator slots are available and how many nodes are intended to be a validator.

Reviewer 2: ```suggestion
If you go to the Staking tab, you will see a list of active validators currently running on the network. At the top of the page, it shows how many validator slots are available as well as how many nodes have signaled their intention to be a validator.
```

> If you go to the Staking tab, you will see a list of active validators currently running on the network. At the top of the page, it shows how many validator slots are available as well as how many nodes have signaled their intention to be a validator.

Reviewer: ```suggestion
If you go to the "Staking" tab, you will see a list of active validators currently running on the network. At the top of the page, it shows how many validator slots are available as well as how many nodes have signaled their intention to be a validator.
```

> You can also go to the Waiting tab to double check to see whether your node is listed there.

Reviewer: ```suggestion
You can also go to the "Waiting" tab to double check to see whether your node is listed there.
```
